src/index.html

```diff
@@ -190,6 +190,37 @@ <h2 class="text-subheader color-gray pb2 sm:pb1_75 md:pb2_5">
           </div>
         </div>
       </section>
+
+      <section
+        class="contact-section flex flex-col sm:flex-row justify-between items-center site-content-max-width inner-content-max-width mxauto contact-page sm:pt4 sm:pb5 site-padding-x"
```

Comment:
the two max-width classes are contradictory here

Reply:
Gooood catch, ty!